packages/devtools_app/lib/src/flutter/banner_messages.dart

```diff
@@ -236,10 +273,10 @@ class DebugModePerformanceMessage {
       key: Key('DebugModePerformanceMessage - $screenType'),
       textSpans: [
         const TextSpan(
-          text:
-              'You are running your app in debug mode. Debug mode performance '
-              'is not indicative of release performance.\n\nRelaunch your '
-              'application with the \'--profile\' argument, or ',
+          text: '''
+You are running your app in debug mode. Debug mode performance is not indicative of release performance.
```

Comment:
does the extra '\n' before  "You are running" 
show up in how the text span is rendered?

Reply:
no it does not